### PR TITLE
New recipe: Odysee

### PR DESCRIPTION
**Pull request by hafiz-muhammad**
Has notification badges. No notification sound provided by the service.

### CODE REVIEW

File: recipes/odysee/webview.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      let unreadNotifications = 0;
+      for (const notificationCounterElement of document.querySelectorAll(
+              ".notification__count"
+          )) {
+          unreadNotifications =
+              unreadNotifications +
+              Ferdium.safeParseInt(notificationCounterElement.textContent);
+      }
+
+      unreadNotifications = unreadNotifications / 1;
```
Comment (vraravam) [inline recipes/odysee/webview.js:12]: What's the point of this division by 1?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/odysee/webview.js (hunk 1 above)
Comment (hafiz-muhammad) [reply recipes/odysee/webview.js]: I apologize about that. I have removed it.

**Review by hafiz-muhammad — COMMENTED** [review]
Removed unnecessary division.

**Review by vraravam — APPROVED** [review]
LGTM.

I dont use this service, so can't validate that the unread badge counter logic is correct.

**Comment by vraravam** [comment]
@all-contributors please add @hafiz-muhammad for code

**Pull request merged**